pkg/maps/tunnel/tunnel.go

```diff
@@ -54,12 +54,18 @@ var (
 			}
 
 			return &k, &v, nil
-		}).WithCache()
+		}).WithCache(),
+	}
 )
 
+// Map is a wrapper around bpf.Map.
```

Comment:
Do you think there's a more useful description we could provide for more context?

I'm thinking something like,

```
Map implements tunnel connectivity configuration in the BPF datapath.
```

Reply:
Fixed in next upload.